src/App.tsx

```diff
@@ -1,9 +1,36 @@
+/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { getTodos } from './api/todos';
+import { wait } from './utils/fetchClient';
+import { getFilteredTodos } from './services/getFilteredTodos';
+
+const USER_ID = 149;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState(Status.All);
+  const [errorMessage, setErrorMessage] = useState('');
+  const filteredTodos = getFilteredTodos(todos, status);
+
+  const itemsLeft = todos.filter(({ completed }) => {
+    return !completed;
+  }).length;
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage('Error reading list todos');
+
+        return wait(3000).then(() => setErrorMessage(''));
```

Comment:
You don`t need to return wait function, just only run it and return nothing and it should work the same way
```suggestion
        wait(3000).then(() => setErrorMessage(''));
```